src/content/blog/2024/office_hours.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: 'Introducing nf-core office hours'
+subtitle: 'A new initiative - weekly drop-in sessions to connect and chat.'
```

Comment:
```suggestion
subtitle: 'A new initiative - weekly drop-in sessions to connect and get help.'
```

Or something? It's a bit more than just chatting

Reply:
connect and work together?

Reply:
```suggestion
subtitle: 'A new initiative - weekly drop-in sessions to connect and work together.'
```